### PR TITLE
Rewrite for speed.

### PR DESCRIPTION
**Pull request by seven1m**

### CODE REVIEW

**Comment by bayleedev** [comment]
Do you mean speed or speed ??

**Comment by seven1m** [comment]
LOL the first one silly

**Comment by bayleedev** [comment]
I don't have access to merge this.

File: Manifest.php [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+		if(!isset($this->out_path)) $this->out_path = $this->outPath();
+		if(!preg_match('/\/$/', $this->out_path)) $this->out_path .= '/';
+	}
+
+	/**
+	 * Compiles all assets in a manifest.
+	 *
+	 * @param boolean $simulate set to true to skip compilation and mangling steps (use this in production)
+	 * @return array if($this->mangle) absolute paths to individual built assets
```
Comment (seven1m) [inline Manifest.php]: Oops, this comment is backward.

**Comment by seven1m** [comment]
Try again @blainesch

**Comment by seven1m** [comment]
Thanks!

**Pull request merged**